tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -2040,12 +2036,12 @@ def value_count(x, value):
 
     if np.isnan(value):
         return np.isnan(x).sum()
-    else:
-        return x[x == value].size
+
+    return x[x == value].size
 
 
 @set_property("fctype", "simple")
-def range_count(x, min, max):
+def range_count(x, min_val, max_val):
```

Comment:
I see your intention: min and max are python keywords, so they should not be used. But the config of the feature extractors is part of our API and changing it is in principle a breaking change, which we should be careful about. 
How about keeping the function parameters the same but renaming it internally so that we do nit have any potential of mixup?

Reply:
> API and changing it is in principle a breaking change,

Yep absolutely see your point - I dont think worth making a breaking change over this - will amend

Reply:
Reverted https://github.com/blue-yonder/tsfresh/pull/1103/commits/1e4293086244e80ce4195ca8f010c68f18690c51